server/lib/jobs.js

```diff
@@ -0,0 +1,113 @@
+const Promise = require('bluebird');
+const request = require('superagent');
+
+const history = require('./history');
+const config = require('./config');
+
+module.exports = {
+  usersImport: (connection, file, storage, token) => {
+    const url = `https://${config('AUTH0_DOMAIN')}/api/v2/jobs/users-imports`;
+
+    return new Promise((resolve, reject) => {
+      request
+        .post(url)
+        .set('accept', 'json')
+        .set('Authorization', `Bearer ${token}`)
+        .field('connection_id', connection)
+        .attach('users', file.path)
+        .end((err, res) => {
+          if (err) {
+            return reject(err);
+          }
+
+          if (res.status >= 200 && res.status <= 300) {
```

Comment:
should status `300` be included?

Reply:
also, might just be personal preference, but i think it would make sense to have all the `reject`s grouped together instead of the resolve splitting between them. maybe something like this?

```
          if (err) {
            return reject(err);
          }

          if (res.status >= 300) {
            return reject((res && res.error) || res);
          }

          const job = {
            id: res.body.id,
            type: 'export'
          };
          return history(job, storage)
              .then(() => resolve(res.body));
```